Tests/SwinjectTests/AssemblerSpec.swift

```diff
@@ -258,7 +258,7 @@ class AssemblerSpec: QuickSpec {
             
             it("can't give entities to parent") {
                 let assembler = Assembler()
-                let childAssembler = Assembler([
+                let childAssembler = Assembler(assemblies: [
```

Comment:
If we change this expression to:
```swift
guard let childAssembler = try? Assembler(assemblies: [
    AnimalAssembly()
], parent: assembler) else { return }
```
I will get segmentation fault.
Xcode Version 8.3.1 (8E1000a)